lci_strategic_plugin/src/lci_strategic_plugin.cpp

```diff
@@ -906,12 +916,12 @@ void LCIStrategicPlugin::planWhenDEPARTING(const cav_srvs::PlanManeuversRequest&
 
   // Identify the lanelets which will be crossed by approach maneuvers lane follow maneuver
   std::vector<lanelet::ConstLanelet> crossed_lanelets =
-      getLaneletsBetweenWithException(current_state.downtrack, intersection_end_downtrack, true, true);
+      getLaneletsBetweenWithException(current_state.downtrack, intersection_end_downtrack, true, false);
 
   // Compose intersection transit maneuver
   resp.new_plan.maneuvers.push_back(composeIntersectionTransitMessage(
       current_state.downtrack, intersection_end_downtrack, current_state.speed, intersection_speed_limit,
-      current_state.stamp, intersection_exit_time, crossed_lanelets.front().id(), crossed_lanelets.back().id()));
+      current_state.stamp, intersection_exit_time, crossed_lanelets.back().id(), crossed_lanelets.back().id()));
```

Comment:
Seems like invalid maneuver setting similar to the others above.

Reply:
Same explanation as before where crossed_lanelets's last element is the intersection lanelet due to inclusive bound. And this lanelet should be added to start_lanelet_id so that intersectionTransitPlugin correctly passes it to the basic_autonomy library.